EXOSIMS/SurveySimulation/multiSS.py

```diff
@@ -0,0 +1,904 @@
+from EXOSIMS.Prototypes.SurveySimulation import SurveySimulation
+import astropy.units as u
+import numpy as np
+import time
+
+
+class multiSS(SurveySimulation):
+    def __init__(
+        self,
+        coeffs=[0.1, 0.2, 6, 2 * np.pi, 1],
+        count=0,
```

Comment:
why are these inputs? count, count_1, and counter_3?

Reply:
these counters are to handle logic for selecting first 2 targets in next target, storing the observation start times and logging results in correct starshade